tests/pbft_manager_test.cpp

```diff
@@ -674,14 +674,17 @@ TEST_F(PbftManagerWithDagCreation, proposed_blocks) {
   ProposedBlocks proposed_blocks(db);
 
   std::map<blk_hash_t, std::shared_ptr<PbftBlock>> blocks;
-  const uint32_t block_count = 100;
   // Create blocks
-  for (uint32_t i = 1; i <= block_count; i++) {
-    std::vector<vote_hash_t> reward_votes_hashes;
-    auto block = std::make_shared<PbftBlock>(blk_hash_t(1), kNullBlockHash, kNullBlockHash, kNullBlockHash, 2, addr_t(),
-                                             dev::KeyPair::create().secret(), std::move(reward_votes_hashes));
-    blocks.insert({block->getBlockHash(), block});
+  for (PbftPeriod period = 1; period <= 3; period++) {
+    for (uint32_t i = 1; i <= 40; i++) {
```

Comment:
why 3 and why 40? maybe it is better to add some constants that will be describing it, or at least some comments